pyaedt/edb.py

```diff
@@ -1060,13 +1060,14 @@ def create_cutout_on_point_list(
         output_aedb_path=None,
         open_cutout_at_end=True,
         nets_to_include=None,
+        include_partial_instances=False,
     ):
         """Create a cutout on a specified shape and save it to a new AEDB file.
 
         Parameters
         ----------
         point_list : list
-            List of points defining the cutout shape.
+            Points list defining the cutout shape.
```

Comment:
In my opinion, the original description is easier to understand.

Reply:
yes. furthermore having twice the word list (type and starting phrase) was causing some warnings in Pycharm to me )